XRTK-Core/Packages/com.xrtk.core/StandardAssets/Shaders/MixedRealityWireframe.shader

```diff
@@ -34,49 +59,38 @@ Shader "Mixed Reality Toolkit/Wireframe"
             // Based on approach described in Shader-Based Wireframe Drawing (2008)
             // http://orbit.dtu.dk/en/publications/id(13e2122d-bec7-48de-beca-03ce6ea1c3f1).html
 
-            struct appdata
+            struct v2g
             {
-                float4 vertex : POSITION;
-                UNITY_VERTEX_INPUT_INSTANCE_ID
+                float4 viewPos : SV_POSITION;
+                UNITY_VERTEX_OUTPUT_STEREO
             };
 
-            struct v2g
+            v2g vert(appdata_base v)
             {
-                float4 projectionSpaceVertex : SV_POSITION;
-                float4 worldSpacePosition : TEXCOORD1;
-                UNITY_VERTEX_OUTPUT_STEREO_EYE_INDEX
-            };
+                UNITY_SETUP_INSTANCE_ID(v);
+                v2g o;
+                o.viewPos = UnityObjectToClipPos(v.vertex);
+                UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(o);
+                return o;
+            }
 
-            // worldSpacePosition is to counteract the effect of perspective-correct interpolation so that the lines
+            // inverseW is to counteract the effect of perspective-correct interpolation so that the lines
             // look the same thickness regardless of their depth in the scene.
             struct g2f
             {
-                float4 projectionSpaceVertex : SV_POSITION;
-                float worldSpacePosition : TEXCOORD0;
+                float4 viewPos : SV_POSITION;
```

Comment:
I like the variable names we had before. They're more descriptive about what's actually going on

Reply:
We can change them, but it's going to hinder support since we are porting the working shader from MRTK.  Although It won't affect the actual operation renaming them, it will simply make maintenance tougher.

Reply:
I don't want straight copy/paste.

If you take any changes, we should be ensuring that it's up to par with our current formatting and standards.

I'd rather just identify what needs to be added fixed, it'll help in understanding how it actually works.

Reply:
Updated property names in Wireframe shader.
Compared Standard shader and found only new additions/updates. Property names were the same